README.md

```diff
@@ -3,65 +3,70 @@
   <span>Consul Telemetry Collector</span>
 </h1>
 
-Consul Telemetry Collector is a lightweight OpenTelemetry collector used to
-collect metrics, logs and traces from a Consul cluster and collect metrics from envoy
-service proxies and export them to HCP or another OTLP compliant endpoint.
-The metric sink is encrypted and authorized by the Consul service mesh.
+Consul Telemetry Collector is a lightweight [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) used to collect metrics from Envoy proxies and export them to HCP and other OTLP compliant metrics endpoints. The [Envoy stats sink](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/metrics/v3/metrics_service.proto#extension-envoy-stat-sinks-metrics-service) is encrypted and authorized by the Consul service mesh.
```

Comment:
this is the main and most important change: we don't collect anything from servers right now. Especially not logs/traces